### ISSUE TITLE
New Moving Average Indicators

### ISSUE THREAD

**Issue by brhurley** [ta4j/ta4j#1225]
**I have checked existing issues and wiki**
- [x ] I could not find similar [issues](https://github.com/ta4j/ta4j/issues?utf8=%E2%9C%93&q=)
- [ x] I could not find a solution in the [wiki](https://ta4j.github.io/ta4j-wiki/) or [faq section](https://ta4j.github.io/ta4j-wiki/FAQ.html)

**Is your feature request related to a problem? Please describe.**
I noticed that several Moving Average indicators are not coded.  The list that I came up with is:

  DMA "Displaced Moving Average" 
  EDSMA "Exponential Displaced Moving Average" 
  JMA "Jurik Moving Average" 
  KiJunV2 "Kihon Moving Average" 
  LSMA "Least Squares Moving Average" 
  McGinley "McGinleys Moving Average" 
  SMAV "Savitzky-Golay Moving Average" 
  SMMA "Smoothed Moving Average" 
  TMA "Triangler Moving Average" 
  VIDYA "Chandes Variable Index Dyanamic Moving Average" 
  VMA "Variable Moving Average" 
  VWMA "Volume Weighted Moving Average" 
  WilderMA "Wilders Moving Average" 

I'm most interested in working with the Jurik Moving Average from what I've read about it so far but would like to work with several of them for some testing.

**Describe the solution you'd like**
I don't mind doing the work to implement I'm just checking to see if you want me to contribute to the codebase for any new moving averages that I create?   I've already read the 'how to contribute' pages and will follow the rules to do so.

**Describe alternatives you've considered**
NA

**Additional context**
NA

**Comment by brhurley** [comment]
@naveedh27  I already have the Wilders working and almost ready to commit.  Just waiting on reply above to see if they want 1 PR or all at once.   But I would be grateful for additional help on this if you have time.

I'll work next on DMA "Displaced Moving Average",  EDSMA "Exponential Displaced Moving Average" and 
JMA "Jurik Moving Average"

**Comment by brhurley** [comment]
@nimo23 Ok, Great.   I'll make a PR for review but leave it open and update as work progresses on the additional MA's

**Comment by brhurley** [comment]
I've completed DMA "Displaced Moving Average", EDSMA "Exponential Displaced Moving Average" and
JMA "Jurik Moving Average", TMA "Triangler Moving Average" so far.

You can look at the draft PR and provide any feedback.

https://github.com/ta4j/ta4j/pull/1226


I'll work on LSMA "Least Squares Moving Average", McGinley "McGinleys Moving Average", SMAV "Savitzky-Golay Moving Average" and SMMA "Smoothed Moving Average" next...

**Comment by brhurley** [comment]
Update: ATMAIndicator is complete and LSMA is almost complete.

**Comment by brhurley** [comment]
LSMA "Least Squares Moving Average", McGinley "McGinleys Moving Average", SMAV "Savitzky-Golay Moving Average" and SMMA "Smoothed Moving Average" is complete

These are the last two to work on besides what @naveedh27 is working on.
KiJunV2 "Kihon Moving Average"
VIDYA "Chandes Variable Index Dyanamic Moving Average"

**Comment by brhurley** [comment]
@naveedh27 How are you doing on the VWMA and VMA?   Hopefully I have the final commit ready on the rest of the MA indicators, it's waiting for another review then it should be good to merge after that.

**Comment by brhurley** [comment]
@naveedh27 I'm created the VWMA and that is in review with a new PR.   I'm working on the VMA now.
